modules/app/webpack.config.js

```diff
@@ -68,20 +69,7 @@ module.exports = {
                     keep_fnames: true
                 }
             })
-        ],
-        splitChunks: {
-            chunks: 'all',
-            cacheGroups: {
-                default: false,
-                defaultVendors: {
-                    test: /[\\/]node_modules[\\/]/,
-                    reuseExistingChunk: true,
-                    minChunks: 2,
-                    priority: -10,
-                    filename: 'js/vendors.main~editor.js'
-                }
-            }
-        }
+        ]
```

Comment:
This deletion now causes this:
![image](https://user-images.githubusercontent.com/11612490/159957552-7b650922-a8d8-4eb7-92b5-bba6544d13b0.png)